### DISCUSSION TITLE
Vercel Audiences not working

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#1000] [Help] [answered]
I am trying to set up Vercel Audience but I cannot see any data coming through.

I have installed the `@vercel/analytics` package, imported it in my `_app.tsx` file and added the ` <Analytics />`  component.

I'm using React 18 and Next 12.3.1

I keep seeing the modal below

<img width="580" alt="new-popup" src="https://user-images.githubusercontent.com/52118069/198268358-2a45a261-c122-431d-b525-6871151bb4f4.png">

Thanks 😄

**Comment by amyegan** [comment]
Hi @deanMuse! Can you share the ID of the deployment you made after adding the analytics code to your app? It would be at the end of the URL for the deployment in your dashboard. For example, `vercel.com/my-account/my-project/6PtVvr3UofEWZNt3X6AY5JgdNf3s` has a deployment ID of `6PtVvr3UofEWZNt3X6AY5JgdNf3s`

**Reply by ghost** [reply]
Hi @amyegan . Thanks for reaching out. the deployment ID is `C5NYniDQmQfES9BhMHcnYDBZPopz`

**Comment by Makisuo** [comment]
Having the same issue

**Comment by allister-grange** [comment]
Likewise, my deployment ID is FQBLqz45dQoUz2uztJq1PpjHUQM5

**Comment by MikevPeeren** [comment]
I am having the same problem, My personal website loads it fine using the new app folder structure but my job project does not load it as it still used pages. Using next v13 btw.

Console displays the following:
Failed to load resource: the server responded with a status of 404 () va/script.js   

Deployment ID: 9o6YzDRDpB54af7nHiKGULmcXVkJ

**Comment by jesperlndqvst** [comment]
I also got the same issue.
Deployment ID: 99Uv9gdMASLGQUKww46axnHHUEY1

**Comment by ghost** [comment]
Hi @amyegan could it be that Vercel Audiences is still in beta ?

**Reply by MikevPeeren** [reply]
Yes it is obviously because it is in Beta and thus still have bugs. It works on my personal site so probably something to do with the Pages directory I assume.

**Comment by amyegan** [comment, marked as answer]
Thank you all for coming here to share these details! As @deanMuse brought up, this is because Audiences is still in beta. I'm highlighting these additional reports with the team so they can get a better picture of what is happening 🙇 

There is an open issue in the analytics repo and the team is currently working on a fix! https://github.com/vercel/analytics/issues/1

**Reply by amyegan** [reply]
A fix was just released. After you redeploy, it should start working. Please let me know if you run into any more problems with this 🙏

**Reply by allister-grange** [reply]
And it works, thank you very much!
